apps/sms/js/thread_ui.js

```diff
@@ -2799,10 +2816,14 @@ var ThreadUI = {
       if (Settings.supportEmailRecipient) {
         items.push({
           l10nId: 'sendMMSToEmail',
-          method: function oMMS(param) {
-            ActivityPicker.sendMessage(param);
+          method: () => {
+            this.navigateToComposer({
+              activity: { number: email }
+            });
           },
-          params: [email]
+          // As we change panel here, we don't want to call 'complete' that
+          // changes the panel as well
+          incomplete: true
```

Comment:
That's the fix for a small group-email thread bug, currently it's impossible to send MMS from participants view if you're in group-email thread - if user tries to do that he's just returned to Thread panel